test/test_transforms_tensor.py

```diff
@@ -728,6 +585,132 @@ def test_center_crop(device):
         scripted_fn.save(os.path.join(tmp_dir, "t_center_crop.pt"))
 
 
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('func, method, out_length', [
+    # test_five_crop
+    ("five_crop", "FiveCrop", 5),
+    # test_ten_crop
+    ("ten_crop", "TenCrop", 10)
+])
+@pytest.mark.parametrize('fn_kwargs', [
+    {"size": (5,)},
+    {"size": [5, ]},
+    {"size": (4, 5)},
+    {"size": [4, 5]}
+])
+def test_x_crop(func, method, out_length, fn_kwargs, device):
+    meth_kwargs = fn_kwargs
```

Comment:
As we only use size, let's just parametrize over 'size' isntead of 'fn_kwargs' and do
meth_kwargs = fn_kwargs = {'size': size}

Reply:
done!